README.md

```diff
@@ -94,13 +100,6 @@ Although we definitely will replace current parser with a parser generator, this
 
 **If you have any thought on this, please join our discussion in [this issue](https://github.com/goby-lang/goby/issues/84). We would love to hear some user's feedback 😁**
 
-
-## TODO & WIP
-
-Checkout this [issue](https://github.com/goby-lang/goby/issues/72) for what we will work on before first release.
-
-Also see [huboard](https://huboard.com/goby-lang/goby)
-
 ## Install
 
 1. You must have Golang installed
```

Comment:
I think people can understand what I'm talking about if they're already reading this line 😅. So I don't think we need to change that.